scrapy/core/downloader/contextfactory.py

```diff
@@ -34,7 +34,7 @@ def from_settings(cls, settings, method=SSL.SSLv23_METHOD, *args, **kwargs):
         tls_ciphers = settings['DOWNLOADER_CLIENT_TLS_CIPHERS']
         return cls(method=method, tls_verbose_logging=tls_verbose_logging, tls_ciphers=tls_ciphers, *args, **kwargs)
 
-    def getCertificateOptions(self):
+    def get_certificate_options(self):
```

Comment:
This follows the base class naming convention.

Reply:
> This follows the base class naming convention.
So, base class naming convention supersedes PEP8 naming convention?